### PR TITLE
feat: update the roadmap to reflect what we're actually working on

### PR DESCRIPTION
**Pull request by tomhobson**
# Description

Someone mentioned in twitter (https://twitter.com/AaronTushabe/status/1496180043780964363) that they love our roadmap, but it hasn't been updated in a while!

So here's a quick update to reflect the work that we've been discussing and would like to do.

# Checklist:

- [ ] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [x] Any dependent changes have already been merged.

### CODE REVIEW

File: content/en/community/roadmap.md [hunk 1]
```diff
@@ -9,28 +9,26 @@ weight: 100
...
 ## Current Work
 
-- [Bring your own vault](https://jenkins-x.io/docs/install-setup/installing/boot/secrets/#external)
-- Granularized permissions
-- OpenShift support
-- [Custom buildpacks](https://jenkins-x.io/docs/create-project/build-packs/#using-custom-build-packs)
-- Terraform cloud orchestration
-- Kubernetes 1.16 compatibility
-- Bring your own container registry
+- Improving our documentation and depreceating any documentation that is now defunct
```
Comment (ankitm123) [inline content/en/community/roadmap.md:15]: typo: `deprecating`

File: content/en/community/roadmap.md [hunk 2]
```diff
@@ -9,28 +9,26 @@ weight: 100
...
+- Kubernetes 1.22 support
+- Making our Terraform and GitOps modules inline with one another
+- Introducting security scanning and container signing
+- Build our community and get more stakeholders and users at the table
```
Comment (ankitm123) [inline content/en/community/roadmap.md:19]: We should also add linting and upgrading to go1.17.
So current work is what we are working on now, right? We should update the board to reflect this. I will move a few tickets.

**Comment by ankitm123** [comment]
Need to rebase with latest master, else pipelines created by this PR will be in pending state.

**Comment by babadofar** [comment]
@tomhobson could you rebase this?

**Pull request closed without merging**